Gruntfile.js

```diff
@@ -13,6 +16,53 @@ grunt.initConfig({
 	},
 	jscs: {
 		all: [ "*.js" ]
+	},
+	sass: {
+		dist: {
+			options: {
+				sourceMap: true,
+				outputStyle: "compressed"
+			},
+			files: [ {
+				expand: true,
+				cwd: "scss",
+				src: [ "*.scss" ],
+				dest: "",
+				ext: ".css"
+			} ]
+		}
+	},
+
+	autoprefixer: {
+		dist: {
+			options: {
+				map: true,
+
+				// Target-specific options go here.
+				// browser-specific info: https://github.com/ai/autoprefixer#browsers
+				browsers: [
+				"> 1%",
+				"last 2 versions",
+				"safari >= 5.1",
+				"ios >= 6.1",
+				"android 2.3",
+				"android >= 4",
+				"ie >= 8"
+				]
+			},
+			src: [
+			"*.css"
+			]
+		}
+	},
+	watch: {
+		sass: {
+			files: [ "scss/**/*.scss" ],
+			tasks: [ "sass", "autoprefixer" ],
```

Comment:
There should be an alias that can be called directly (when not using watch), that could be used here as well, like "build".